### ISSUE TITLE
Trouble with "Create a horizontal Line using HR element" lesson

### ISSUE THREAD

**Issue by dkreidler** [freeCodeCamp/freeCodeCamp#18067]
In the [Create a Horizontal Line using HR Element](https://learn.freecodecamp.org/responsive-web-design/applied-visual-design/create-a-horizontal-line-using-the-hr-element) lesson, the check fails when the code is correctly formatted following the instructions. 

Campers are presented with the following code:
```
`<div class="fullCard">
  <div class="cardContent">
    <div class="cardText">
      <h4><s>Google</s>Alphabet</h4>
      
      <p><em>Google was founded by Larry Page and Sergey Brin while they were <u>Ph.D. students</u> at <strong>Stanford University</strong>.</em></p>
    </div>`
```

The instructions are just to add a horizontal rule tag beneath the `<h4>` line. Simple enough. However, the tests won't pass unless you first swap the `<em>` and `<p>` tags on the following paragraph. The test seemingly just has a typo in the code its checking against. 

![fcc hr error](https://user-images.githubusercontent.com/19493034/44891022-15354d80-acab-11e8-829c-294262aa60f1.PNG)

* Browser Name: Chrome
* Browser Version: Version 68.0.3440.106 (Official Build) (64-bit) (up to date)
* Operating System: Windows 10

@QuincyLarson I saw your request that someone create this issue, and didn't see it in the list, so hopefully I'm not duplicating this request... and after two weeks, hopefully I'm not the first reporting it here!

**Comment by dkreidler** [comment]
Seems like it should be an easy enough fix... superficially. I'd like to tackle it, if so... if someone could point me to the right directory(s). Thanks!

**Comment by iambk** [comment]
@dkreidler Sorry, but this issue already exists in both freeCodeCamp (#18030 ) and the curriculum repos. Its actually fixed and merged in the curriculum. But just not live yet. For a workaround with this issue, interchange the <p> and <em> tags.

Next time, feel free to drop in a message in the [Contributers chat room](https://gitter.im/FreeCodeCamp/Contributors) to confirm about an issue.

**Comment by Nirajn2311** [comment]
@dkreidler Could you close this issue as it is a duplicate of #18030

**Comment by dkreidler** [comment]
Closed as duplicate of #18030
